.gitignore

```diff
@@ -77,6 +77,8 @@ packages/gatsby-**/**/*.json
 # these ones are manually added
 !packages/gatsby-**/src/typings/**/*.d.ts
 !packages/gatsby-**/src/**/*.json
+!packages/gatsby-**/package.json
+!packages/gatsby-**/tsconfig.json
```

Comment:
why?

Reply:
I needed to add them here because they were being ignored... probably due to this line: 

```
packages/gatsby-**/**/*.json
```